src/Bicep.Core.UnitTests/Utils/ContainerRegistryClientFactoryExtensions.cs

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Bicep.Core.Registry;
+using Bicep.Core.UnitTests.Registry;
+
+namespace Bicep.Core.UnitTests.Utils;
+
+public class ContainerRegistryClientFactoryExtensions
```

Comment:
these don't appear to be extensions.  Helper?

Reply:
sure, thanks for the tip :)